Content.Server/Disposal/Unit/EntitySystems/DisposalUnitSystem.cs

```diff
@@ -85,7 +84,7 @@ public override void Initialize()
             SubscribeLocalEvent<DisposalUnitComponent, SharedDisposalUnitComponent.UiButtonPressedMessage>(OnUiButtonPressed);
         }
 
-        private void AddDisposalAltVerbs(EntityUid uid, DisposalUnitComponent component, GetVerbsEvent<AlternativeVerb> args)
+        private void AddDisposalAltVerbs(EntityUid disposalUnit, DisposalUnitComponent component, GetVerbsEvent<AlternativeVerb> args)
```

Comment:
this is more of a nitpick that applies to this file but normally we keep the EntityUid variable name as `uid` for consistency